pkg/config/config_parser.go

```diff
@@ -74,6 +71,17 @@ func (p *Parser) decodeConfig(body hcl.Body, diags hcl.Diagnostics) (*Config, hc
 			continue
 		}
 	}
+
+	if hasPolicyBlock {
+		diags = append(diags,
+			&hcl.Diagnostic{
+				Severity: hcl.DiagWarning,
+				Summary:  "Deprecated 'policy' block in config file",
+				Detail:   "Specifying 'policy' blocks in 'config.hcl' has been deprecated. See https://docs.cloudquery.io/docs/tutorials/policies/writing-your-first-policy for instructions on running a local policy.",
+			},
+		)
+	}
```

Comment:
why not append the diag in the switch case?

Reply:
Don't want 10 different diags in case the user has more then one policy block (one is enough to let them know of the deprecation)